MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -491,6 +493,21 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func getHostEnvironmentInfo(with callbackId: String) {
+        hostAppMessageDelegate?.getHostEnvironmentInfo(completionHandler: { (result) in
+            switch result {
+            case .success(let response):
+                guard let encodedResult = ResponseEncoder.encode(data: response) else {
+                    self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.internalError))
+                    return
+                }
+                self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: encodedResult)
+            case .failure(let error):
+                self.handleMASDKError(error: error, callbackId: callbackId)
```

Comment:
`handleMASDKError` internally uses `getMiniAppErrorMessage`. Can you create a similar method that actually calls `prepareMAJavascriptError`